### PR TITLE
Move tests

### LINKED ISSUE #1802: Move automated tests to main repository
To be able to run the automated tests easily as part of our dev cycle (e.g. when submitting PR to review), the tests should be part of the platform repository instead apinf-testing repo.

Proposed workflow for issues before submitting PR's to review
1. Lint you code
2. Make sure any database migration is taken care of
3. Merge conflicts
4. Run automated tests
5. Mark you PR as 'ready to review'
# Definition of done
- [x] Move the JS tests to our main repository to /test
  ' [ ] If possible, try to preserve the commit history
- [x] Set up an NPM alias to run tests, e.g. npm test

### PR DESCRIPTION
**Pull request by jykae**
Closes #1802 

## Proposed changes
- adds new folder "test" + guide
- pulled tests from https://github.com/apinf/apinf-testing with well preserved commits
- "npm test" definition to package.json to run tests

### CODE REVIEW

File: test/common.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    url: 'https://nightly.apinf.io',
+    buildDriver: function() {
+        var driver = new webdriver.Builder().withCapabilities(webdriver.Capabilities.chrome()).build();
+        driver.manage().timeouts().setScriptTimeout(5000);
+        driver.manage().timeouts().implicitlyWait(5000);
+        return driver;
+    },
+    signIn: function(driver) {
+        driver.get(this.url);
+        driver.findElement(By.linkText('login')).click();
```
Comment (marla-singer) [inline test/common.js]: @jykae We will have problem in this place because now navbar has 'Sign in' button for login.

Comment (jykae) [inline test/common.js]: @marla-singer well noticed, and that's what tests are good for, they fail if something is different. :)

**Comment by jykae** [comment]
Ran the whole test set:
nayttokuva 2016-10-31 kello 12 35 56

**Comment by marla-singer** [comment]
@jykae 8.1 test doesn't work as expected because we have an e-mail verification on nightly

**Comment by jykae** [comment]
@marla-singer I ran this one against localhost as noticed email verifications are ON nightly.

Changing url of instance we are testing against should be simpler, as well as the login user that is expected to be "robot@test.com". We might need to come up with decision what is basically the instance we want to run these tests. If it's nightly we need to turn off email verifications OFF for registration tests.

Basically here the task was to move tests & make it possible to run them using "npm test".

**Comment by jykae** [comment]
@marla-singer assign yourself if you would like to review this one. I propose that fixing tests would be in separate PRs, if something is needed.

**Comment by jykae** [comment]
Github login above failed possibly because I had settings for Github wrong in localhost..

**Comment by marla-singer** [comment]
@jykae Merge it.

**Comment by brylie** [comment]
:electric_plug: :green_apple: :boat:
